capella2polarion/converters/converter_config.py

```diff
@@ -165,14 +168,39 @@ def set_diagram_config(self, diagram_config: dict[str, t.Any]):
         p_type = diagram_config.get("polarion_type") or "diagram"
         self.polarion_types.add(p_type)
         links = _filter_links(
-            c_type, _force_link_config(diagram_config.get("links", []))
+            c_type, self._force_link_config(diagram_config.get("links", []))
         )
         self.diagram_config = CapellaTypeConfig(
-            p_type,
+            add_prefix(p_type, self.type_prefix),
             diagram_config.get("serializer") or "diagram",
             links + self._get_global_links(c_type),
         )
 
+    def _force_link_config(self, links: t.Any) -> list[LinkConfig]:
+        result: list[LinkConfig] = []
+        for link in links:
+            if isinstance(link, str):
+                config = LinkConfig(
+                    capella_attr=link,
+                    polarion_role=add_prefix(link, self.role_prefix),
+                )
+            elif isinstance(link, dict):
+                config = LinkConfig(
+                    capella_attr=(lid := link["capella_attr"]),
+                    polarion_role=add_prefix(
+                        link.get("polarion_role", lid), self.role_prefix
+                    ),
```

Comment:
I would recommend to add additional attributes `link_field` and `reverse_field` to the `LinkConfig`. These will then be used as custom field IDs in the LinkConverter.
```suggestion
                    polarion_role=add_prefix(
                        (pid := link.get("polarion_role", lid)), self.role_prefix
                    ),
                    link_field=(lf := link.get("link_field", pid)),
                    reverse_field=link.get("reverse_field", f"{lf}_reverse"),
```